boards/arm/gd32e103v_eval/doc/index.rst

```diff
@@ -0,0 +1,212 @@
+.. _gd32e103v_eval:
+
+GigaDevice GD32E103V-EVAL
+#########################
+
+
+Overview
+********
+
+The GD32E103V-EVAL board is a hardware platform that enables design and debug
+of the GigaDevice C103V Cortex-M4F High Performance MCU.()
```

Comment:
`.()`? C103V or E103V?

Reply:
remove `()` and change `C103V` to `E103`